app/services/database_service.py

```
@@ -423,126 +390,203 @@ def list_content_items(
+        try:
+            with get_session() as session:
+                # media_url typically contains the full YouTube URL
+                obj = (
+                    session.query(Transcript)
+                    .filter(
+                        Transcript.media_url.ilike(f"%{video_id}%")
+                    )
+                    .first()
+                )
+                return obj.to_dict() if obj else None
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)